src/metabase/lib/metadata/calculation.cljc

```diff
@@ -451,17 +451,19 @@
    ReturnedColumnsOptions
    [:map
     ;; these all default to true
-    [:include-joined?              {:optional true} :boolean]
-    [:include-expressions?         {:optional true} :boolean]
-    [:include-implicitly-joinable? {:optional true} :boolean]]])
+    [:include-joined?                              {:optional true} :boolean]
+    [:include-expressions?                         {:optional true} :boolean]
+    [:include-implicitly-joinable?                 {:optional true} :boolean]
+    [:include-implicitly-joinable-for-source-card? {:optional true} :boolean]]])
```

Comment:
I think this flag points to some inconsistency in the product. Apparently, in some places (e.g., viz settings) we want to see implicitly joinable fields for _source cards_, in other places (orderable and breakoutable columns) we don't.

Reply:
@bshepherdson see @luizarakaki's  [comment](https://github.com/metabase/metabase/issues/30950#issuecomment-1567139855) related to this.